package.json

```diff
@@ -17,6 +17,7 @@
     "lint:ci-deactivated": "eslint -f checkstyle -o reports/checkstyle.xml  .",
     "build": "npm pack && gunzip -qf -c linkurious-lke-plugin-data-table-$(cat .version).tgz > lke-plugin-data-table.lke",
     "plugin:tar": "scripts/postpack.js",
+    "upload": "gh release create \"v$(cat .version)\" lke-plugin-data-table.lke",
```

Comment:
@enys is the name `upload` reserved for something ? Can we call it `publish:github` to bring more clarity ?

Reply:
It is reserved by the CI yes. It's generic. We could add another scritpt and then call it here, but not sure of the value.